app/src/main/java/org/simple/clinic/summary/medicalhistory/MedicalHistorySummaryView.kt

```diff
@@ -100,12 +97,10 @@ class MedicalHistorySummaryView(
         updatedAtDisplayText
     )
 
-    diagnosedForHypertensionQuestionView.render(DIAGNOSED_WITH_HYPERTENSION, medicalHistory.diagnosedWithHypertension, ::answerToggled)
-    treatmentForHypertensionQuestionView.render(IS_ON_TREATMENT_FOR_HYPERTENSION, medicalHistory.isOnTreatmentForHypertension, ::answerToggled)
     heartAttackQuestionView.render(HAS_HAD_A_HEART_ATTACK, medicalHistory.hasHadHeartAttack, ::answerToggled)
     strokeQuestionView.render(HAS_HAD_A_STROKE, medicalHistory.hasHadStroke, ::answerToggled)
     kidneyDiseaseQuestionView.render(HAS_HAD_A_KIDNEY_DISEASE, medicalHistory.hasHadKidneyDisease, ::answerToggled)
-    diabetesQuestionView.render(HAS_DIABETES, medicalHistory.hasDiabetes, ::answerToggled)
+    diabetesQuestionView.render(MedicalHistoryQuestion.HAS_DIABETES, medicalHistory.hasDiabetes, ::answerToggled)
```

Comment:
Can you add import for `MedicalHistoryQuestion.HAS_DIABETES`? so that it's same the above ones?

Reply:
Done.